src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -73,7 +73,8 @@ struct
   include VarMetadataTbl (VM)
   open VM
 
-  let local x = make_var ~name:x.vname Local
+  let local_name x = x.vname ^ "#" ^ string_of_int(x.vid)
```

Comment:
Or even now that we use different names for them, in the above `VM` module they could have a `varinfo` argument, because the correspondence is one-to-one. That would cause a quite widespread change though (like simplifying `to_cil_varinfo` below to be direct), so maybe we hold off that refactoring from this PR.